src/pe/exception.rs

```diff
@@ -587,13 +587,13 @@ impl<'a> UnwindInfo<'a> {
         // whenever flags UNW_FLAG_EHANDLER or UNW_FLAG_UHANDLER are set. The language-specific
         // handler is called as part of the search for an exception handler or as part of an unwind.
         } else if flags & (UNW_FLAG_EHANDLER | UNW_FLAG_UHANDLER) != 0 {
-            let offset = bytes.gread_with::<u32>(&mut offset, scroll::LE)? as usize;
+            let address = bytes.gread_with::<u32>(&mut offset, scroll::LE)?;
             let data = &bytes[offset..];
```

Comment:
should data use `offset` or `address` here?

Reply:
Thanks for asking, data should in fact use `offset`. 

For posterity: This is documented [here](https://docs.microsoft.com/en-us/cpp/build/exception-handling-x64?view=vs-2019#struct-unwind_info). With the handler flags set, `struct UNWIND_INFO` contains a `u32` address of the exception handler, followed immediately by variable length data.